app-android-xamarin/Moodis/Moodis/Feature/Menu/MenuActivity.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Support.V7.App;
+using Android.Util;
+using Android.Widget;
+using Java.Lang;
+using Moodis.Feature.CameraFeature;
+using Moodis.Ui;
+
+namespace Moodis.Feature.Menu
+{
+    [Activity(Label = "Menu Activity")]
+    public class MenuActivity : AppCompatActivity
+    {
+        private const string WarningInRequest = "Something wrong happened. Is your internet turned on?";
```

Comment:
Move these strings to strings.xml, and fetch them with `this.getString(R.string.warning_face_detection`